examples/client/client.c

```diff
@@ -1323,6 +1409,31 @@ THREAD_RETURN WOLFSSL_THREAD client_test(void* args)
         }
     #endif
 
+    #ifdef WOLFSSL_TLS13
+    if (!helloRetry) {
+        if (onlyKeyShare == 0 || onlyKeyShare == 1) {
+        #ifdef HAVE_FFDHE_2048
+            if (wolfSSL_UseKeyShare(ssl, WOLFSSL_FFDHE_2048) != SSL_SUCCESS) {
+                err_sys("unable to use DH 2048-bit parameters");
+            }
+        #endif
+        }
+        if (onlyKeyShare == 0 || onlyKeyShare == 2) {
+            if (wolfSSL_UseKeyShare(ssl, WOLFSSL_ECC_SECP256R1)
+                    != SSL_SUCCESS) {
+                err_sys("unable to use curve secp256r1");
+            }
+            if (wolfSSL_UseKeyShare(ssl, WOLFSSL_ECC_SECP384R1)
+                    != SSL_SUCCESS) {
+                err_sys("unable to use curve secp384r1");
+            }
+        }
+    }
+
+    if (noPskDheKe)
+        wolfSSL_no_dhe_psk(ssl);
```

Comment:
Should there be a wolfSSL_CTX_ version of no_dhe_psk as well to make this easier to use?